frontend/src/App.tsx

```diff
@@ -1,12 +1,14 @@
 import React from 'react';
+import Header from './components/shared/header/Header';
 import Routes from './routes/Routes';
 
 function App() {
-  return (
-    <div className="App">
-      <Routes />
-    </div>
-  );
+	return (
```

Comment:
Please check your indentation settings - it seems that there is not 2 but 6 spaces by default